asteroid/dsp/beamforming.py

```diff
@@ -439,3 +524,4 @@ def _precision_mapping():
 BeamFormer = Beamformer
 SdwMwfBeamformer = SDWMWFBeamformer
 MvdrBeamformer = RTFMVDRBeamformer
+GevdBeamformer = GEVDBeamformer
```

Comment:
The aliases above were just not to break backward compat, no need to add it for new classes.

Reply:
Ok